_gsocproposals/2018/proposal_HAhRD3D-Clustering.md

```diff
@@ -1,9 +1,12 @@
 ---
 title: 3D-Clustering
 layout: gsoc_proposal
-project: HAhRD
+project: 
+  - CMS
+  - HAhRD
```

Comment:
A proposal can be supported by multiple organizations, but only one project. Can you choose either CMS or HAhRD?